tailcfg/tailcfg.go

```diff
@@ -73,7 +73,8 @@ type CapabilityVersion int
 //	36: 2022-08-02: added PeersChangedPatch.{Key,DiscoKey,Online,LastSeen,KeyExpiry,Capabilities}
 //	37: 2022-08-09: added Debug.{SetForceBackgroundSTUN,SetRandomizeClientPort}; Debug are sticky
 //	38: 2022-08-11: added PingRequest.URLIsNoise
-const CurrentCapabilityVersion CapabilityVersion = 38
+//  39: 2022-08-15: clients can talk Noise over arbitrary HTTPS port
```

Comment:
gofmt? (with 1.19)

This looks like a tabs/spaces mismatch, at least here on GitHub reviews